package.json

```diff
@@ -86,19 +86,19 @@
   "dependencies": {
     "@ministryofjustice/frontend": "^1.2.0",
     "agentkeepalive": "^4.2.0",
-    "applicationinsights": "^2.2.0",
+    "applicationinsights": "^2.2.1",
     "body-parser": "^1.19.1",
     "bunyan": "^1.8.15",
     "bunyan-format": "^0.2.1",
     "compression": "^1.7.4",
     "connect-flash": "^0.1.1",
-    "connect-redis": "^6.0.0",
+    "connect-redis": "^6.1.1",
     "cookie-session": "^2.0.0",
     "csurf": "^1.11.0",
-    "dotenv": "^14.3.0",
+    "dotenv": "^16.0.0",
```

Comment:
only effects values starting with **#** or containing **`**. Don't believe we have either 👍

Reply:
changelog: https://github.com/motdotla/dotenv/blob/master/CHANGELOG.md